app/src/main/java/io/left/reflect/MainActivity.java

```diff
@@ -38,15 +39,9 @@
  * </p>
  */
 public class MainActivity extends AppCompatActivity implements MeshStateListener,
-        RightMeshRecipientComponent.RecipientChangedListener {
+        RightMeshRecipientView.RecipientChangedListener{
     private static final String TAG = MainActivity.class.getCanonicalName();
 
-    /**
-     * MESH_PORT is the mesh port that this app is allowed to run on, according to your license key.
-     * See developer.rightmesh.io for more details.
-     */
-    private static final int MESH_PORT = 9876;
```

Comment:
With the change in mesh port, has someone updated the licenses? Todo on the internal side - we may want to have a set of meshports in the dev portal which map to all of the github projects and when someone creates a new account they automatically get access to all of these mesh ports.